test/Microsoft.DocAsCode.Build.Engine.Tests/DocumentBuilderTest.cs

```diff
@@ -994,8 +994,8 @@ public async Task TestBuildWithXrefService()
                 StatusCode = HttpStatusCode.OK,
                 Content = new StringContent("[{'uid':'csharp_coding_standards', 'name':'C# Coding Standards', 'href':'http://dotnet.github.io/docfx/guideline/csharp_coding_standards.html'}]")
             });
-
-            var httpClient = new HttpClient(fakeResponseHandler);
+            // this method only used for unit test.
```

Comment:
remove this comment since this is a test class already

Reply:
done